src/goto-instrument/cover.cpp

```diff
@@ -1277,8 +1226,74 @@ void instrument_cover_goals(
   const symbol_tablet &symbol_table,
   goto_programt &goto_program,
   coverage_criteriont criterion,
-  coverage_goalst &goals)
+  bool function_only)
+{
+  coverage_goalst goals; //empty already covered goals
+  instrument_cover_goals(symbol_table,goto_program,
+                        criterion,goals,function_only,false);
+}
+
+/*******************************************************************\
+
+Function: consider_goals
+
+  Inputs:
+
+ Outputs:
+
+ Purpose:
+
+\*******************************************************************/
+
+bool consider_goals(const goto_programt &goto_program)
+{
+  bool result;
```

Comment:
please initialize, this might simplify setting of `result` below, i.e., `result = case1 || case2` instead of `if(...) ... else ...`